source/sync/configure/enable-sync.txt

```diff
@@ -219,5 +213,21 @@ Procedure
                curl https://services.cloud.mongodb.com/api/admin/v3.0/groups/{GROUP_ID}/apps/{APP_ID}/services/{MongoDB_Service_ID}/config \
                  -X PATCH \
                  -h 'Authorization: Bearer <Valid Access Token>' \
-                 -h "Content-Type: application/json"
-                 -d @/sync/config.json
+                 -h "Content-Type: application/json" \
+                 -d '<Flexible Sync Configuration>'
+
+            .. code-block:: json
+
+               {
+                 "flexible_sync": {
+                   "state": "enabled",
+                   "database_name": "<Name of Database>",
+                   "client_max_offline_days": <Number>,
+                   "is_recovery_mode_disabled": <Boolean>,
+                   "indexed_queryable_fields_names": <Array of String Field Names>,
+                   "queryable_fields_names": <Array of String Field Names>,
+                   "collection_queryable_fields_names": <Map[String][]String>
```

Comment:
```suggestion
                   "indexed_queryable_fields_names": [ <Array of String Field Names> ],
                   "queryable_fields_names": [ <Array of String Field Names> ],
                   "collection_queryable_fields_names": { <Map[String][String]> }
```

Reply:
Don't think is necessary, as text already says array and map.